sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_create_and_deploy_project.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import pytest
+
+from azure.core.exceptions import HttpResponseError, ClientAuthenticationError
+from azure.core.credentials import AzureKeyCredential
+
+from testcase import (
+    QuestionAnsweringTest,
+    GlobalQuestionAnsweringAccountPreparer,
+    QnaAuthoringHelper
+)
+
+from azure.ai.language.questionanswering.projects import QuestionAnsweringProjectsClient
+
+class CreateAndDeployTests(QuestionAnsweringTest):
+
+    @GlobalQuestionAnsweringAccountPreparer()
+    def test_create_project(self, qna_account, qna_key):
```

Comment:
Will these tests be recorded?

Reply:
yes, was just waiting for your review
didn't wanna add unnecessary extra files until i resolve all comments and make sure all tests are passing